torch/_torch_docs.py

```diff
@@ -4942,6 +4970,34 @@ def merge_dicts(*dicts):
             [-0.0760, -3.6705,  2.4784]])
 """.format(**common_args, **tf32_notes))
 
+add_docstr(torch.hspmm,
```

Comment:
How is this function distinct from torch.sparse.smm or torch.mm?

Reply:
`torch.hspmm` is distinct in returning a (1+1)-D hybrid sparse tensor, while `torch.smm` returns a 2-D (non-hybrid) sparse tensor and `torch.mm` returns a strided tensor.